src/standard.rs

```diff
@@ -43,11 +50,26 @@ pub struct EncryptedDocument {
     pub document: HashMap<FieldId, EncryptedBytes>,
 }
 
+#[derive(Debug, Clone, uniffi::Record)]
+pub struct RekeyEdeksBatchResult {
```

Comment:
If there's no non-batch it might not be necessary to include `Batch` in the type, just like it isn't included in the function name. I could understand the value in including it to try to give people an expectation that anything with `BatchResult` in the name will have the same interface though... I don't know which is more useful.

Reply:
I think the `BatchResult` is useful to indicate the consistent result type